src/App.tsx

```diff
@@ -146,41 +131,35 @@ export const App: React.FC = () => {
         {isSplashScreen ? (
           <SplashScreen />
         ) : (
-          <><div className="reset_orbit_selection_container">
-              <div
-                className="reset_selection top"
-                onClick={resetOrbitSelection}
-              ></div>
-              <div
-                className="reset_selection bottom"
-                onClick={resetOrbitSelection}
-              ></div>
-            </div>
-            <div className="nav_container">
-                {!flyToId && (
-                  <div className="orbit_title_container">
-                    {orbitIds.map((id, index) => {
-                      return (
-                        <OrbitTitleGroup
-                          key={id}
-                          id={id}
-                          tabIndex={index} />
-                      );
-                    })}
-
-                    <div className="popup_line_container">
-                      {orbitIds.map((id) => {
-                        return <PopUpLine key={id + "_popupline"} id={id} />;
-                      })}
-                    </div>
-                  </div>
-                )}
-
-                {!flyToId && activeId && (
-                  <PopUpText />
-                )}
-              </div></>
+          <div className="reset_orbit_selection_container">
+            <div
+              className="reset_selection top"
+              onClick={resetOrbitSelection}
+            ></div>
+            <div
+              className="reset_selection bottom"
+              onClick={resetOrbitSelection}
+            ></div>
+          </div>
+        )}
+        {showNavBar && (
+          <div className="nav_container">
+            {!flyToId && (
+              <div className="orbit_title_container">
+                {orbitIds.map((id, index) => {
+                  return <OrbitTitleGroup key={id} id={id} tabIndex={index} />;
+                })}
+
+                <div className="popup_line_container">
+                  {orbitIds.map((id) => {
+                    return <PopUpLine key={id + "_popupline"} id={id} />;
+                  })}
+                </div>
+              </div>
+            )}
```

Comment:
Whether to show the nav is independent of the splash screen.
In mobile the nav is visible on the splash screen.